### DISCUSSION TITLE
How do I get "index.js" to run?

### DISCUSSION THREAD

**Discussion by Istionia** [javascripteverywhere/api#19] [Q&A]
So, I tried to run the "index.js" file on my Terminal using the "node src" command. Here was the result:

`Timothy@Arthurs-MacBook-Air-2 ~ % node src/index.js
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/Timothy/src/index.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
`
I expected to be able to open a browser window to http://localhost:4000 & see the file displayed on it. How, then, do I solve this problem?

**Comment by Raikyo777** [comment]
Friend, you must be in terminal not in root folder  `Timothy@Arthurs-MacBook-Air-2 ~ %
Example: 
cd /Folder_where_you_download_NOTEDLY_Project
cd /api
and only after then press node src/index.js
